composer.json

```diff
@@ -4,7 +4,7 @@
     "description": "A Sylius plugin for handling brands",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": ">=7.2",
```

Comment:
It's maybe better to completely remove this line, because the PHP version is already handle by Sylius itself, WDYT ?